### ISSUE TITLE
Unknown HTTP error

### ISSUE THREAD

**Issue by brianjmurrell** [dequis/purple-facebook#307]
I seem to be able to get connected for a while and send messages but after a while I will get disconnected with:

```
<account-name> disconnected
Unknown HTTP error
```

Trying to reconnect doesn't seem to work.  It just says the above again.

I'm running https://github.com/dequis/purple-facebook/releases/tag/v0.9.2-c9b74a765767

Any ideas?

**Comment by dequis** [comment]
Debug log please https://github.com/dequis/purple-facebook/wiki/Debugging-Logs

**Comment by brianjmurrell** [comment]
Interestingly I came back after a while and I was connected again.

I will try to grab a debug log the next time I see it happening.

Is the `PURPLE_UNSAFE_DEBUG=1 PURPLE_VERBOSE_DEBUG=1 pidgin --debug` any different from enabling debug in the UI with _Help->Debug Window_?

**Comment by dequis** [comment]
Yeah, this plugin doesn't show anything useful in debug without those because it has private stuff all over the place.

**Comment by deltaflyer4747** [comment]
Now it atleast logs in, however still : 
 Null value for $.viewer.message_threads.sync_sequence_id

**Comment by brianjmurrell** [comment]
Heh.  Now I am also getting:

```
Null value for $.viewer.message_threads.sync_sequence_id
```

**Comment by dequis** [comment]
@brianjmurrell 0.9.3?
